packages/Webkul/Shop/src/Http/Controllers/Customer/AddressController.php

```diff
@@ -67,19 +60,19 @@ public function store(AddressRequest $request)
     {
         $customer = auth()->guard('customer')->user();
 
+        // $requestInput = request()->only('_token','company_name' , 'first_name' , 'last_name' , 'vat_id' , 'address1' , 'country' , 'state' , 'city' , 'postcode' , 'phone');
+
+        $request->mergeRequest($customer);
```

Comment:
Make $data variable here,

~~~php
$data = [
    'company_name' => '',
    'first_name'          => '',
    'last_name'          => '',
    ...
];
~~~

Reply:
Or you can simply pass this full array in the repository,

~~~php
$this->customerAddressRepository->create([
    'company_name' => '',
    'first_name'          => '',
    'last_name'          => '',
    ...
]);
~~~

Reply:
fixed